test/start-stop.node.js

```diff
@@ -244,7 +264,7 @@ types.forEach((type) => {
         expect(ipfsd).to.exist()
       })
 
-      it('daemon should not be running', (done) => {
+      it('daemon should be running', (done) => {
```

Comment:
Why this change?

Reply:
It was misnamed, we are explicitly initing and starting the node.